install/kubernetes/sidecar-template.yaml

```diff
@@ -0,0 +1,90 @@
+initContainers:
+- name: istio-init
+  image: "gcr.io/istio-testing/proxy_init:3101ea9d82a5f83b699c2d3245b371a19fa6bef4"  
+  args: ["-p","15001", "-u", "1337"]
+  imagePullPolicy: "IfNotPresent"
+  securityContext:
+    capabilities:
+      add:
+      - NET_ADMIN
+    priviledged: true
```

Comment:
please put a guard around this. this is a temporary workaround, and should not be enabled unless required.